lib/lita/handlers/ai.rb

```diff
@@ -11,6 +11,7 @@ def chat(payload)
         return unless chatting?(payload[:message])
         message = extract_aliases(payload[:message])
         reply = self.class.cleverbot.think(message.body)
+        reply = reply.to_s.split('|').map { |s| s.length == 4 ? [s.hex].pack("U") : s }.join('') if reply.to_s[0] == '|'
```

Comment:
Would the first character always be a `|`, or could a reply contain mixed unicode and normal text? I'm guessing things like emojis might also come back as unicode, so it might be worth re-writing this to match any 4 characters preceded by `|`?

Looking at the docs for `hex` (I haven't done much with unicode before!), it'll return 0 for no match. We could also use this to avoid incorrect matches. Who know's what cleverbot might reply with..